src/maincurrencydialog.cpp

```diff
@@ -432,8 +432,8 @@ void mmMainCurrencyDialog::OnListItemSelected(wxDataViewEvent& event)
                 else if (Model_CurrencyHistory::instance().find(Model_CurrencyHistory::CURRENCYID(m_currency_id)).size() > 0)
                 {
                     if (wxMessageBox(wxString::Format(_(
-                        "Historic rates for %1$s found, but \"Use currency history\" in options is disabled:\n"
-                        "click no and enable it or click yes to remove all historic rates for %2$s"),
+                        "Historical rates for %1$s found, but “Use historical currency” in options is disabled:\n"
```

Comment:
I'm a bit surprised the removal of the escape character here didn't cause any issues.

Reply:
Does the smart quote require an escape character?

Reply:
Ah, I didn't see the standard `"` character was replaced with the left/right quotation mark unicode symbols. Makes sense.